scripts/ubuntu/openstack.sh

```diff
@@ -0,0 +1,10 @@
+apt-get -y install pump cloud-utils cloud-init cloud-initramfs-growroot \
+    bash-completion
+
+# Ignore wardning about datasources
+mkdir -p /etc/cloud/cloud.cfg.d
+cat <<EOF >/etc/cloud/cloud.cfg.d/99-warnings.cfg;
+#cloud-config
+warnings:
+  dsid_missing_source: off
+EOF
```

Comment:
Without this file, anytime you log into an instance you get a scary warning about the instance only being able to find Amazon EC2 datasources, and not openstack datasources that we have it configured to use.